modules/campaignion_overlay/campaignion_overlay.module

```diff
@@ -6,24 +6,27 @@
  */
 
 /**
- * Implements hook_theme().
```

Comment:
Now there is no theme function, right? What I was aiming for was **one** theme function that includes all the field_collection and renders all the fields.